config/settings.yml

```diff
@@ -422,9 +422,6 @@ adapted_housing:
 intent_to_file:
   prefill: true
 
-ivc_champva:
-  prefill: true
-
```

Comment:
I think we still need this for prefill to work in general

Reply:
So just leave this file as it was originally basically? Or should I remove 7959f-1 from line 1347?

Reply:
Yeah I don't think you need to do anything in settings. I don't see "7959f-1" anywhere in settings.yml let alone line 1347

Reply:
Ok cool!